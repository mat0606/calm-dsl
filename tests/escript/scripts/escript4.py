# python3;success

print("python3")
output = urlreq("https://pypi.org/pypi/sampleproject/json", verb="GET")
email = output.json()["info"]["author_email"]

pprint(email)
print(uuid.uuid4() == uuid.uuid4())

import requests

req_out = requests.get("https://pypi.org/pypi/sampleproject/json")
print(req_out.text == output.text)
