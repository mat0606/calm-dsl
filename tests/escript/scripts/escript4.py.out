python3
u'"A. Random Developer" <author@example.com>'
False
True
